$OpenBSD$

Don't build the help subdir.
Help files uses some old textproc system last released
over a decade ago.

--- Makefile.am.orig	Sat Sep 27 22:23:16 2014
+++ Makefile.am	Sat Sep 27 22:23:22 2014
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = help man pixmaps po scripts src
+SUBDIRS = man pixmaps po scripts src
 
 desktopdir = $(datadir)/applications/
 desktop_in_files = gnome-hearts.desktop.in
